exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -62,7 +62,7 @@ public int HowManyPlayersInGame()
             return totalPlayers.Count;
         }
 
-        public void Roll(int roll)
+        public void RollDice(int roll)
```

Comment:
Much easier to understand what the method does without having to actually read the code in the method.